EventFinder-DDLs.sql

```diff
@@ -0,0 +1,208 @@
+DROP TABLE DoB;
+DROP TABLE User;
+DROP TABLE CreditCard;
+DROP TABLE Organizer;
+DROP TABLE Location;
+DROP TABLE Event;
+DROP TABLE PaidEvent;
+DROP TABLE Staff;
+DROP TABLE Company;
+DROP TABLE Sponsor;
+DROP TABLE EventType;
+DROP TABLE WorkOn;
+DROP TABLE EventHasType;
+DROP TABLE InterestedIn;
+DROP TABLE Friend;
+DROP TABLE PayWith;
+DROP TABLE Participate;
+
+CREATE TABLE DoB (
+    DateOfBirth DATE PRIMARY KEY,
+    Age INT NOT NULL
+);
+
+CREATE TABLE User (
+    UserID INT PRIMARY KEY AUTO_INCREMENT,
+    FirstName VARCHAR(50) NOT NULL,
+    LastName VARCHAR(50) NOT NULL,
+    DateOfBirth DATE NOT NULL,
+    Gender VARCHAR(50) NOT NULL,
+    FOREIGN KEY (DateOfBirth) REFERENCES DoB(DateOfBirth) ON DELETE NO ACTION ON UPDATE CASCADE
+);
+
+CREATE TABLE CreditCard (
+    CardNumber VARCHAR(16) PRIMARY KEY,
+    ExpiryDate DATE NOT NULL,
+    HolderName VARCHAR(100) NOT NULL,
+    CVC INT NOT NULL
+);
+
+CREATE TABLE Organizer (
+    OrganizerUserID INT PRIMARY KEY,
+    OrganizerContactInfo VARCHAR(200) NOT NULL,
+	FOREIGN KEY (OrganizerUserID) REFERENCES User(UserID) ON DELETE CASCADE
+);
+
+CREATE TABLE Location (
+    Address VARCHAR(200) PRIMARY KEY,
+    Name VARCHAR(200) NOT NULL

```

Comment:
Location does not have name anymore. Let's just completely remove the location table and all its foreign keys.